src/content/packages-and-plugins/using-packages.md

```diff
@@ -124,10 +124,10 @@ To add the package, `css_colors`, to an app:
 
 ### Adding a package dependency to an app using `flutter pub add`
 
-To add the package, `css_colors`, to an app:
+To add the package `css_colors` to an app:
 
 1. Issue the command while being inside the project directory
```

Comment:
Consider specifying what "the command" is. It's a good opportunity to cross-link to the `pub add` docs as well.

```
1. Use the [`pub add`][] command from inside the project directory

...

[`pub add`]: {{site.dart-site}}/tools/pub/cmd/pub-add
```